Added SparklinesInteractiveLayer

vegetableman opened the pull request:
Made it a little interactive

![interactive](https://user-images.githubusercontent.com/1330851/44942688-5164cd00-add5-11e8-9f66-5dc99029aee1.gif)

arslanhashmi commented:
@borisyankov are we going to have this interactive layer at some point?

qcho commented:
Great PR! Just what I was looking for.

Question. Did you need an updated version of `react` and `react-dom` for this?

It's probably making the scope for this PR bigger than it should

vegetableman commented:
It's been 2 years... time sure passes.

Anyways, @qcho had to check my code again. On the issue, I updated `react` because I was using `React.Fragment` which is not supported in react 15. I have removed fragment, reverted the versions back to old one and now am using the svg  `g` tag which works well.

Cheers!
